gemrb/core/Scriptable/Actor.cpp

```diff
@@ -6233,6 +6233,12 @@ bool Actor::ValidTarget(int ga_flags, const Scriptable *checker) const
 			if (Modified[IE_CHECKFORBERSERK]) return false;
 		}
 	}
+	if (ga_flags & GA_BUMPING) {
```

Comment:
Doesn't handle a null checker.

Reply:
Please also rename the bit — it's now not clear if you mean bumpable or able of bumping.